hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: minor
+
+This release adds a new module, ``hypothesis.extra.grammar``, which you
```

Comment:
I think maybe we want to think about whether we really want a single `grammar` extra namespace here - we're likely to want to support other things and we don't want to force people to depend on all of them just to use one of them.